ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -92,12 +92,23 @@ def pytest_addoption(parser):
         help="Collect OCS logs when test case failed",
     )
     parser.addoption(
-        '--io_in_bg',
+        '--io-in-bg',
         dest='io_in_bg',
         action="store_true",
         default=False,
         help="Run IO in the background",
     )
+    parser.addoption(
+        '--io-load',
```

Comment:
please update readme docs on when to use and when not to use and maybe one or two usecase.

Reply:
good idea! will add